### PR TITLE
 Fixed issue #19397: Editing menu entries not possible

### PR DESCRIPTION
**Pull request by Shnoulle**
Dev: fix tab to space in view
Dev: parent are not tr, in body
Dev: add data-menuentryid on HTML
Dev: Fix delete on same way

### CODE REVIEW

File: application/controllers/admin/SurveymenuEntryController.php [hunk 1]
```diff
@@ -76,11 +76,18 @@ public function getsurveymenuentryform($menuentryid = null)
         $menuentryid = Yii::app()->request->getParam('menuentryid', null);
         if ($menuentryid != null) {
             $model = SurveymenuEntries::model()->findByPk(((int) $menuentryid));
+            if (empty($model)) {
+                throw new CHttpException(404, gT("This menuentries don't exist."));
```
Comment (tiborpacalat) [inline application/controllers/admin/SurveymenuEntryController.php:80]: "menuentries" should be "menu entries"

File: application/controllers/admin/SurveymenuEntryController.php [hunk 2]
```diff
@@ -119,12 +126,15 @@ public function update($id)
...
         //Update or create
-        $id = (int) $id;
+        $id = intval($id);
         if ($id != 0) {
             $model = SurveymenuEntries::model()->findByPk($id);
         } else {
             $model = new SurveymenuEntries();
         }
+        if (empty($model)) {
+            throw new CHttpException(404, gT("Invalid menu entrie."));
```
Comment (tiborpacalat) [inline application/controllers/admin/SurveymenuEntryController.php:136]: "entrie" should be "entry".

**Review by tiborpacalat — CHANGES_REQUESTED** [review]

File: application/controllers/admin/SurveymenuEntryController.php (hunk 1 above)
Comment (Shnoulle) [reply application/controllers/admin/SurveymenuEntryController.php:80]: "menu entry" single :)

File: application/controllers/admin/SurveymenuEntryController.php (hunk 2 above)
Comment (Shnoulle) [reply application/controllers/admin/SurveymenuEntryController.php]: _Invalid menu entry_. for both

**Review by gabrieljenik — APPROVED** [review]
Code structure looks ok

**Review by gabrieljenik — APPROVED** [review]
Code structure looks OK

**Pull request merged**